### PR TITLE
Update resources for soc v2

### PR DESCRIPTION
**Pull request by ls-yannick**
Updates the terraform to be compatible with the SOC v2.

### CODE REVIEW

File: ground_stations/main.tf [hunk 1]
```diff
@@ -21,12 +21,16 @@ resource "leanspace_nodes" "ground_stations" {
     key   = "country"
     value = each.value.country
   }
-  tags {
-    key   = "LEAFSPACE-ID"
-    value = each.value.leafspaceId
-  }
+}
+
+resource "leanspace_leaf_space_ground_station_links" "ground_station_link" {
```
Comment (jerome-leanspace) [inline ground_stations/main.tf:26]: I guess we should make it depend on the connection of leaf space so it doesn't try to do this before the connection is set up

File: main.tf [hunk 2]
```diff
@@ -25,6 +25,18 @@ module "plugins" {
   source = "./satellites/plugins"
 }
 
+resource "leanspace_leaf_space_integrations" "integration_Connection" {
```
Comment (jerome-leanspace) [inline main.tf:28]:
```suggestion
resource "leanspace_leaf_space_integrations" "integration_connection" {
```

File: service_accounts/test-release-orchestrator/main.tf
Comment (jerome-leanspace) [inline service_accounts/test-release-orchestrator/main.tf:1]: we don't need this?

File: satellites/command_queues/main.tf
Comment (jerome-leanspace) [inline satellites/command_queues/main.tf:1]: We have to tell customers that they have to "clean" the queue before (i.e.: [delete transmissions](https://docs.leanspace.io/docs/api/commands%2Foperations%2Fdelete-a-command-queue-transmission))

File: states/plan/main.tf
Comment (jerome-leanspace) [inline states/plan/main.tf:1]: We have to tell customers that they have to delete the plans using these states

File: service_accounts/test-release-orchestrator/main.tf
Comment (ls-yannick) [reply service_accounts/test-release-orchestrator/main.tf:1]: Indeed, the file is deleted :)

File: ground_stations/main.tf (hunk 1 above)
Comment (ls-yannick) [reply ground_stations/main.tf:26]: I'm not sure how we can accomplish this, any suggestions?

File: satellites/command_queues/main.tf
Comment (ls-yannick) [reply satellites/command_queues/main.tf:1]: [Added a note about it in the migration guide](https://github.com/leanspace/docs-space-operations-center/pull/16/files#diff-17e3a6789cb4307c3205125a656d6f625c3d25d8c09e84c41bbfb523a611dc51R27).

File: service_accounts/test-release-orchestrator/main.tf
Comment (jerome-leanspace) [reply service_accounts/test-release-orchestrator/main.tf:1]: Yes but I meant how is the release orchestrator doing its tests without the service account used to get access to Leanspace
Or was this simply moved to another part?

Comment (ls-yannick) [reply service_accounts/test-release-orchestrator/main.tf:1]: Defering this questions to @FlorianLacour

File: ground_stations/main.tf (hunk 1 above)
Comment (jerome-leanspace) [reply ground_stations/main.tf:26]:
```
"depends_on": ["resource.leanspace_leaf_space_integrations.integration_connection"]
```

File: satellites/command_queues/main.tf
Comment (jerome-leanspace) [reply satellites/command_queues/main.tf:1]: Actually I heard back from Vivien that you don't need to do this

File: ground_stations/main.tf (hunk 1 above)
Comment (ls-yannick) [reply ground_stations/main.tf:26]: THanks!

File: satellites/command_queues/main.tf
Comment (ls-yannick) [reply satellites/command_queues/main.tf:1]: Docs updated!

File: service_accounts/test-release-orchestrator/main.tf
Comment (FlorianLacour) [reply service_accounts/test-release-orchestrator/main.tf:1]: I thought that as it was for houstontwo we will not create it. 
The ROC have test on dev, I don't know if we want to keep it in main :/

Comment (jerome-leanspace) [reply service_accounts/test-release-orchestrator/main.tf:1]: Right, it's ok to only have it on dev :)

**Pull request merged**